polynote-frontend/style/styles.less

```diff
@@ -2147,6 +2147,7 @@ button {
 
 .notebook-cells {
   position: relative;
+  margin-bottom: 20em; // so that the last cell isn't at the very bottom of the page (eg so context menu displays nicely)
```

Comment:
It's not smart enough to make the menu expand upward if you're at the bottom of the page?

Reply:
It tries to be smart but has this annoying behavior when at the bottom of the screen where it gets very small. Compare this: 

![Screen Shot 2019-07-08 at 10 24 13 AM](https://user-images.githubusercontent.com/5430417/60829805-a1858c00-a16a-11e9-8a21-17b286baf80a.png)

with:

![Screen Shot 2019-07-08 at 10 24 40 AM](https://user-images.githubusercontent.com/5430417/60829809-a64a4000-a16a-11e9-8d14-0d9e1db001bd.png)

Reply:
Again, its possible there's some trick here I missed that would make it work :)